Modules/xStorage/DSCResources/MSFT_xDiskAccessPath/MSFT_xDiskAccessPath.psm1

```diff
@@ -524,7 +511,7 @@ function Test-TargetResource
     {
         Write-Verbose -Message ( @(
                 "$($MyInvocation.MyCommand): "
-                $($LocalizedData.DiskNotGPTMessage -f $DiskNumber,$Disk.PartitionStyle)
+                $($localizedData.DiskNotGPTMessage -f $DiskNumber,$Disk.PartitionStyle)
```

Comment:
@TravisEz13 - sorry about that - I was actually working on the HQRM for this but once the DSCResource.Tests went through it broke my HQRM PR so I ended up combining them. Again, sorry about that.